launchable/commands/record/tests.py

```diff
@@ -442,3 +449,19 @@ def get_session_and_record_start_at_from_subsetting_id(subsetting_id: str, clien
         "session": "builds/{}/test_sessions/{}".format(build_number, test_session_id),
         "start_at": parse_launchable_timeformat(created_at)
     }
+
+
+def get_env_values(client: LaunchableClient) -> Dict[str, str]:
+    sub_path = "slack/notification/key/list"
+    res = client.request("get", sub_path=sub_path)
+
+    metadata = {}  # type: Dict[str, str]
+    if res.status_code != 200:
```

Comment:
[question]
While you are using `res.raise_for_status()` [here](https://github.com/launchableinc/cli/pull/423/files#diff-a54ea2cd0c54426ceb96ffb93873062a86aaa7d4d4cb1310ab874ed0eeda2e85R335), why are you validating status code != 200 here?

Reply:
Good question, I think notifying slack is an optional feature. So I don't want to suspend main feature (record tests & close session) so I don't use raise_for_status when we request to `slack/notification/key/list`

Reply:
Good.